### PR TITLE
Payload stats

### PR DESCRIPTION
**Pull request by rahulbhivare**

### CODE REVIEW

**Review by bcwaldon — COMMENTED** [review]
Hi @rahulbhivare, overall this is in the right direction. I left a few comments in the code for you. I would also suggest you use the term "metric" instead of "stat" or "statsd". The word "metric" is more general, while "statistics" might imply more analysis is happening and "statsd" is a specific tool/implementation. We can discuss this as users defining "payload metrics" where each "metric" is a counter, gauge, etc.

File: lib/python/satos_payload_sdk/app_framework.py [hunk 1]
```diff
@@ -252,6 +253,21 @@ def _handle_response(self, params):
...
+    def _handle_payload_stats(self, params):
+        print("Inside _handle_payload_stats")
+        return self._req_payload_stats_cb(params)
+    
+class PayloadStatsdInfo:
+    def __init__(self):
+        self.stats_counter = 0            # Counter number
+        self.stats_names = [''] * 32      # Counter names, string
+
+class Payload:
```
[the remark this answers is not in the document]
Comment (rahulbhivare) [reply lib/python/satos_payload_sdk/app_framework.py]: Updated

File: lib/python/satos_payload_sdk/app_framework.py [hunk 2]
```diff
@@ -252,6 +253,21 @@ def _handle_response(self, params):
     def _handle_start_sequence(self, params):
         return self._start_sequence_cb(params.sequence_id, params.sequence_params, params.scheduled_deadline)
 
+    def _handle_payload_stats(self, params):
+        print("Inside _handle_payload_stats")
+        return self._req_payload_stats_cb(params)
+    
+class PayloadStatsdInfo:
```
[the remark this answers is not in the document]
Comment (rahulbhivare) [reply lib/python/satos_payload_sdk/app_framework.py]: Deleted class. Thank you for pointing out. I missed that point.

File: lib/python/satos_payload_sdk/app_framework.py [hunk 3]
```diff
@@ -371,3 +390,20 @@ def _handle_shutdown(self, params):
         self.request_stop()
 
         return api_types.AntarisReturnCode.An_SUCCESS
+
+    def _req_payload_stats(self, params):
```
[the remark this answers is not in the document]
Comment (rahulbhivare) [reply lib/python/satos_payload_sdk/app_framework.py]: Added code to reset counter values to 0.

**Pull request merged**